cmd/prometheustosfx/main.go

```diff
@@ -10,6 +10,7 @@ import (
 	//"strconv"
```

Comment:
Not sure what you've meant here. Confused with line numbers to which this comment assigned.